tools/validate-xep0001-conformance.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# Checks if provided file (filename to be provided as first argument) conforms to XEP-0001.
+# See https://github.com/xsf/xeps/issues/1235
+#
+# Expected to be executed from the directory that holds all xep XML files.
+# Requires one argument: the file name of the xep to be validated, eg:
+#
+# $ tools/validate-xep0001-conformance.sh xep-0010.xml
+#
+# exit status will be non-zero upon validation failure.
+#
+# requires: bash, xmllint
+
+set -e
+
+VALIDATION_RESULT=0
+
+# 1. Check DTD conformance against xep.dtd
+XEP_DTD="xep.dtd"
+if (xmllint --noout --dtdvalid "$XEP_DTD" "$1")
```

Comment:
You do not need `()` around the command in an `if`. I think that causes a subshell to spawn, the effects of which I'm not certain about. Scripts are arcane.